### PR TITLE
Page not found

### LINKED ISSUE #44: Doctor card keep loading if URL is invalid
Eg: https://stage-zdravniki.sledilnik.org/sl/zdravnik/neveljavno-ime

should show a not found (possibly 404 to prevent page from being indexed) and offer the user a link to search page with preselected doctor type

### PR DESCRIPTION
**Pull request by miaerbus**
Partially fixes #44

@stefanb what exactly did you mean by "offer the user a link to search page with preselected doctor type". Did you mean a link to the homepage with preselected doctor type filter (maybe as a query param) or a additional search field in the 404 page or something else?

@bananica maybe a different image for 404 page, what do you think?

### CODE REVIEW

**Comment by stefanb** [comment]
Just a plain 404 page in needed, but should be rarely seen by users

**Review by jalezi — APPROVED** [review]

File: src/components/SEO/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+export const Static = function Static({ children }) {
+  return (
+    <Helmet>
+      <html lang="en" />
+      <title>Hello from React Helmet</title>
+      <meta name="description" content="Basic example" />
+      {children}
+    </Helmet>
+  );
+};
```
Comment (stefanb) [inline src/components/SEO/index.js:12]: Do we need this with such values?

Comment (jalezi) [reply src/components/SEO/index.js:12]: No. Sorry, we don't need Static component at the moment. I wasn't sure what I will use and forgot to delete it.

Comment (stefanb) [reply src/components/SEO/index.js:12]:
```suggestion
```

**Comment by stefanb** [comment]
Some translations are missing:
image

Please check the JSON structure (`pageNotFound` vs `SEO.title` groups)

`/sl/eNotFound` comes from language switching, "/sl/" overwriting start of "/PageNotFound" URL.

**Comment by miaerbus** [comment]
Possible fix for the problem https://github.com/sledilnik/zdravniki/commit/2068d2d56036be5edd6f1961f2578be8effcd0ad, can I merge it to this branch?

**Comment by stefanb** [comment]
@miaerbus sure, whatever is needed. A separatee `404` branch on it's own makes no sense.

**Comment by bananica** [comment]
@miaerbus I'll try to come up with a new image/illustration for 404, I love interesting 404 pages, 404 doctor not found ;)

**Comment by miaerbus** [comment]
Added fix for #58

**Comment by miaerbus** [comment]
- Fixed an issue that always redirected to 404 on doctor page.
- Switched to `useDoctors` hook because `useFilter` was returning only GP doctors, dentists and gynaecologists were missing.
- I think 'react-router-dom' package is enough, maybe we can remove 'react-router'

**Pull request merged**